### ISSUE TITLE
Attributes in a proof request aren't focusable or read using a screen reader

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1479]
### Description of problem
In a proof request, with voice over on, the credential as a whole is focused but the individual attributes aren't read out. For someone who has any visual impairment, they won't know what the proof request is asking.

### Expected behavior
Attributes should be read with a screen reader

### Steps to reproduce
- turn on voice over on iOS
- receive a proof request
- swipe through until the focus lands on the credentials requested

### Environment
- [ ] Occurs on Android
- [X] Occurs on iOS

Build #: 1203

Android Device Model: 

iOS Device Model: iPhone SE

### Severity
- [ ] High
- [X] Medium
- [ ] Low

**Comment by knguyenBC** [comment]
Checked again and it works fine.
